scripts/common/sonic-stp-build/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -xe
+
+#install libevent_2
+sudo apt-get -y purge libevent-dev
+sudo dpkg -i target/debs/stretch/libevent_2.*.deb
```

Comment:
The file does not exist, see: https://sonic-jenkins.westus2.cloudapp.azure.com/job/vs/job/buildimage-vs-all/lastSuccessfulBuild/artifact/target/debs/stretch/

Reply:
Libevent version 2.1.8 was added as part of STP feature and the same will be available once below PR is merged.
https://github.com/Azure/sonic-buildimage/pull/3463/